drivers/lora/sx126x.c

```diff
@@ -433,7 +433,7 @@ static int sx126x_lora_init(const struct device *dev)
 	const struct sx126x_config *config = dev->config;
 	int ret;
 
-	LOG_DBG("Initializing %s", DT_INST_LABEL(0));
+	LOG_DBG("Initializing sx126x");
```

Comment:
Why not `LOG_DBG("Initializing %s", dev->name);`? 
I know the driver currently isn't multi-instance, but it seems more standard to me.

Reply:
figured it was a DBG message so not that critical to use `dev->name` (and if we phase out name/make it optional in the future its one less thing to fix/change).

Reply:
I'd have problems with phasing out names (I have boards with 36 instances of the same sensor).
Anyway, I don't have strong opinions for this case.